apps/web/app/future/settings/(admin)/admin/page.tsx

```diff
@@ -1,13 +1,10 @@
-import LegacyPage from "@pages/settings/admin/index";
 import { _generateMetadata } from "app/_utils";
-import { WithLayout } from "app/layoutHOC";
-
-import { getLayout } from "@components/auth/layouts/AdminLayoutAppDir";
 
 export const generateMetadata = async () =>
   await _generateMetadata(
     () => "Admin",
     () => "admin_description"
   );
 
-export default WithLayout({ getServerLayout: getLayout, Page: LegacyPage })<"P">;
```

Comment:
Are we missing something from here - page/layout seems to be missing in the new changes

Reply:
(admin) route group has a layout file, so all the nested pages inside (admin) route group will get passed the layout and page wrapper